src/components/Players/PlayerManager.vue

```diff
@@ -341,6 +338,14 @@ export default Vue.extend({
         }
       }, this.getOsdTimeoutDuration());
     },
+    toggleMute(): void {
+      if (this.currentVolume !== 0) {
+        this.previousVolume = this.currentVolume;
+        this.setVolume({ volume: 0 });
+      } else {
+        this.setVolume({ volume: this.previousVolume });
+      }
+    },
```

Comment:
As we're duplicating logic from VolumeSlider, what I'd do is just set this as an action in the store. That way we can also store previousVolume there and factorize the logic in this component and VolumeSlider.